src/Control/Effect/Cull.hs

```diff
@@ -14,12 +14,11 @@ module Control.Effect.Cull
 , run
 ) where
 
-import Control.Applicative (Alternative(..))
```

Comment:
This is also being exported from `Control.Effect.NonDet`, which we import below. ghc 8.8.1 is smarter about catching redundant imports like this.